frontend/src/components/new-position-form.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { moment } from "moment";
+
+const toNames = ({ first_name, last_name }) => `${first_name} ${last_name}`;
+
+const validNumber = val => (val && isNaN(val) ? "Not a valid number" : false);
+const validDate = val =>
+    (val && moment(val, "YYYY-MM-DD", true).isValid()) || val === ""
+        ? false
+        : "Not a valid date";
+// check for format XXX123H1
+const validCourseCode = val => {
```

Comment:
No. Don't do this check. The backend will say whether a course code is valid. If it exists in the database, it's valid.

Reply:
Removed check